website/docs/cli/commands/plan.html.md

```diff
@@ -189,6 +191,79 @@ module, aside from the `-var` and `-var-file` options. For more information,
 see
 [Assigning Values to Root Module Variables](/docs/language/values/variables.html#assigning-values-to-root-module-variables).
 
+### Input Variables on the Command Line
+
+You can use the `-var` command line option to specify values for
+[input variables](/docs/language/values/variables.html) declared in your
+root module.
+
+However, to do so will require writing a command line that is parsable both
+by your chosen command line shell _and_ Terraform, which can be complicated
+for expressions involving lots of quotes and escape sequences. In most cases
+we recommend using the `-var-file` option instead, and write your actual values
+in a separate file so that Terraform can parse them directly, rather than
+interpreting the result of your shell's parsing.
+
+To use `-var` on a Unix-style shell on a system like Linux or macOS we
+recommend writing the option argument in single quotes `'` to ensure the
+shell will interpret the value literally:
+
+```
+terraform plan -var 'name=value'
+```
+
+If your intended value also includes a single quote then you'll still need to
+escape that for correct interpretation by your shell, which also requires
+temporarily ending the quoted sequence so that the backslash escape character
+will be significant:
+
+```
+terraform plan -var 'name=va'\''lue'
+```
+
+When using Terraform on Windows, we recommend using the Windows Command Prompt
+(`cmd.exe`). When you pass a variable value to Terraform from the Windows
+Command Prompt, use double quotes `"` around the argument:
+
+```
+terraform plan -var "name=value"
```

Comment:
Do you want to use `terraform.exe` in the windows-specific examples? (I don't have strong feelings about this, just a ? 🤷 ).

Reply:
:thinking: I guess I don't typically type the `.exe` when I'm running commands on Windows (since Windows searches it by default) so my inclination is not to include it, but I dunno if my practices generalize to others. I'll do a little research and see what some similar products show in their Windows examples, assuming I can find any that _have_ Windows-specific examples. :upside_down_face: 

Thanks!

Reply:
I did a little quick research by searching for various "How to use X on Windows" tutorials, where X included both Terraform and some other similar command-line-driven software, and all of the top search results I found showed command names without the `.exe` (or other) suffix, so I think I'm going to run with it as I wrote it here and then watch out for any questions in the usual places that might suggest surprise or confusion about this and adjust if needed. Thanks again for pointing that out!